ui/views.py

```diff
@@ -136,6 +137,16 @@ def __call__(self, request, repo_slug):
         if repo_slug not in set([x.slug for x in repos]):
             return HttpResponseForbidden("unauthorized")
         self.repo = [x for x in repos if x.slug == repo_slug][0]
+        # get sorting from params if it's there
+        params = dict(request.GET.copy())
+        if ('sortby' in params and
```

Comment:
Also, `dict` will create a copy itself so wouldn't `request.GET.copy` be redundant? If the `pop` were done to `params` it wouldn't modify the original request (which I agree would be sketchy)